tests/conftest.py

```diff
@@ -19,9 +19,8 @@
 # default timout threshold is 2 seconds for determinign long and short
 DEFAULT_TIMEOUT = 2.0
 
-# Enforce use of deterministic kernels
-# see composer.utils.reproducibility.configure_deterministic_mode
-os.environ["CUBLAS_WORKSPACE_CONFIG"] = ":4096:8"
```

Comment:
This was introduced in https://github.com/mosaicml/composer/commit/79d0bec0b9e2fdf22837ef1632d81c8b03ee883e

based on the note in: https://docs.nvidia.com/cuda/cublas/index.html#cublasApi_reproducibility

I don't see it added back in for `configure_deterministic_mode`. Was this supposed to have been removed?

Reply:
`configure_deterministic_mode` sets this variable. See https://github.com/mosaicml/composer/blob/dc920e3c13d03b33a4b7d2443cfc7fb7e31f9d2f/composer/utils/reproducibility.py#L116